laa_crime_application_store_app/routers/application.py

```diff
@@ -83,3 +83,31 @@ async def post_application(request: ApplicationNew, db: Session = Depends(get_db
         nested.rollback()
         return Response(status_code=409)
     return Response(status_code=201)
+
+
+@router.put("/application/{app_id}", status_code=201, responses=responses)
+async def put_application(
+    app_id: UUID, request: ApplicationNew, db: Session = Depends(get_db)
+):
+    application = db.query(Application).filter(Application.id == app_id).first()
+    if application is None:
+        logger.info("APPLICATION_NOT_FOUND", application_id=app_id)
+        return Response(status_code=400)
```

Comment:
If there is no application, this should be considered and upsert and therefor create a new entry - https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT

Reply:
then what is the point of the post endpoint...

Reply:
This code is still reading as if there isnt an application then we are returning a 400. The update should still have the risk value as it can be amended by the caseworker.